src/main/java/no/entur/uttu/routing/RoutingService.java

```diff
@@ -0,0 +1,16 @@
+package no.entur.uttu.routing;
+
+import java.math.BigDecimal;
+
+/**
+ * For getting the road geometry - resulting in service links and links in sequence in a journey pattern
+ */
+public interface RoutingService {
```

Comment:
--> RoutingService<Mode>

bean -> OsrmBusRoutingService implements RoutingService<Bus>

ServiceLinkContext -> mode
ServiceLinkProducer decides which bean to call for a given mode